AmplifyPlugins/Auth/AWSAuthPlugin/Dependency/AuthorizationProviderAdapter.swift

```diff
@@ -37,4 +38,31 @@ class AuthorizationProviderAdapter: AuthorizationProviderBehavior {
     func invalidateCachedTemporaryCredentials() {
         awsMobileClient.invalidateCachedTemporaryCredentials()
     }
+
+    private func setupListener() {
+        awsMobileClient.addUserStateListener(self) { [weak self] state, _ in
+            guard let self = self else {
+                return
+            }
+            switch state {
+            case .signedOutFederatedTokensInvalid,
+                 .signedOutUserPoolsTokenInvalid:
+                print("AWSMobileClient Event listener - \(state)")
```

Comment:
Here and throughout: Use `Amplify.Logging`, not `print()`

Reply:
Changed